src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -626,17 +631,17 @@ real64 CompositionalMultiphaseBase::updatePhaseVolumeFraction( ObjectManagerBase
   return maxDeltaPhaseVolFrac;
 }
 
-void CompositionalMultiphaseBase::updateFluidModel( ObjectManagerBase & dataGroup ) const
+void CompositionalMultiphaseBase::updateFluidModel( ElementSubRegionBase & subRegion ) const
```

Comment:
I would have kept this an `ObjectManagerBase` though coz the method could work to update these fields on a `FaceManger` for example.